cloudinit/config/cc_growpart.py

```diff
@@ -316,6 +373,23 @@ def resize_devices(resizer, devices):
                          "failed to resize: disk=%s, ptnum=%s: %s" %
                          (disk, ptnum, e),))
 
+        if is_lvm and isinstance(resizer, ResizeGrowPart):
+            try:
+                if len(devices) == 1:
+                    (_out, _err) = subp.subp(
+                        ["lvm", "lvextend", "--extents=100%FREE", blockdev],
+                        update_env={'LANG': 'C'})
+                    info.append((devent, RESIZE.CHANGED,
+                                 "Logical Volume %s extended" % devices[0],))
+                else:
+                    util.logexc(LOG, "Exactly one device should be configured "
+                                     "to be resized when using LVM. More than "
+                                     "one configured: %s" % devices)
```

Comment:
and I'm pretty sure that stack traces are WARNings, so I'd really like this to be at strongest an INFO message.